tdrs-backend/tdpservice/search_indexes/management/commands/clean_and_reparse.py

```diff
@@ -323,10 +345,16 @@ def handle(self, *args, **options):
 
         all_fy = "All"
         all_q = "Q1-4"
-        log(f"Starting clean and reparse command for FY {fiscal_year if fiscal_year else all_fy} and "
-            f"{fiscal_quarter if fiscal_quarter else all_q}",
-            logger_context=log_context,
-            level='info')
+
+        if not selected_files:
+            log(f"Starting clean and reparse command for FY {fiscal_year if fiscal_year else all_fy} and "
+                f"{fiscal_quarter if fiscal_quarter else all_q}",
+                logger_context=log_context,
+                level='info')
+        else:
+            log(f"Starting clean and reparse action for files: {str(selected_files)}",
```

Comment:
@jtimpe I am confirming that this log change works as expected when triggering reparsing from DAC and terminal on a specific files 👍🏾

Reply:
T = reparsing from terminal
D = reparsing from DAC

![actions_and_commands](https://github.com/user-attachments/assets/ea40fa27-2222-4e94-9dd5-bb1f85c6c877)